libs/portals/my-pages/health/src/screens/OrganDonation/components/RegistrationForm.tsx

```diff
@@ -87,23 +97,32 @@ export const Form2 = () => {
         input: {
           isDonor: radioValue === OPT_IN || radioValue === OPT_IN_EXCEPTIONS,
           organLimitations: radioValue === OPT_IN_EXCEPTIONS ? limitations : [],
+          comment: comment as string,
```

Comment:
is this safe?

Reply:
No, I forgot about this. This was typed as an object and then the api changed to string 😬 Will fix!